src/lib/components/SearchBar.svelte

```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-  import { goto } from "$app/navigation";
```

Comment:
At some point we should add eslint warnings for these "untranslated" imports (goto and redirect). There are already similar custom ESLint rules in place so it should be quite easy.

Reply:
Good idea!